gamestonk_terminal/forex/fx_view.py

```diff
@@ -233,7 +233,7 @@ def create_order(accountID, instrument, other_args: List[str]):
         "--price",
         dest="price",
         action="store",
-        type=float,
+        type=check_non_negative,
```

Comment:
and now you can remove the one below in line 248! 

Thank you! Sorry for being picky :b